### DISCUSSION TITLE
push rtmp failed :  closed (unable to parse H264 config: invalid size 1)

### DISCUSSION THREAD

**Discussion by hdlk510** [bluenviron/mediamtx#1069] [Questions]
start the container with:
`docker run  -it --name rtsmp_Server --network=host aler9/rtsp-simple-server`
>root@cnbjs01-rd-qa01:/home/bingo# docker run  -it --name rtsmp_Server --network=host aler9/rtsp-simple-server
2022/08/05 08:19:46 INF rtsp-simple-server v0.19.3
2022/08/05 08:19:46 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2022/08/05 08:19:46 INF [RTMP] listener opened on :1935
2022/08/05 08:19:46 INF [HLS] listener opened on :8888
2022/08/05 08:19:55 INF [RTSP] [conn 10.100.105.8:34778] opened
2022/08/05 08:19:55 INF [RTSP] [session 317917922] created by 10.100.105.8:34778
2022/08/05 08:19:55 INF [RTSP] [session 317917922] is publishing to path 'test110', 1 track with UDP

The RTSP works well,but when I push rtmp, not work, the output of server is:
>2022/08/05 08:20:15 INF [RTMP] [conn 10.100.105.26:51550] opened
2022/08/05 08:20:15 INF [RTMP] [conn 10.100.105.26:51550] closed (unable to parse H264 config: invalid size 1)

and the output of client is:
>WriteN, RTMP send error 32 (129 bytes)
WriteN, RTMP send error 32 (38 bytes)
WriteN, RTMP send error 9 (42 bytes)
av_interleaved_write_frame(): Operation not permitted
Receive NULL pic0.0 q=-0.0 size=     128kB time=00:00:00.03 bitrate=30895.1kbits/s speed= 113x
[flv @ 0x19d44e0] Failed to update header with correct duration.
[flv @ 0x19d44e0] Failed to update header with correct filesize.
Error writing trailer of rtmp://10.10.1.55/live/abc: Operation not permitted

Where is wrong, any help?
Thanks.

**Comment by aler9** [comment]
Please open an issue and provide relevant infos in order to replicate the bug, thanks.
